docs/UserGuide/Maintenance-Tools/Metric-Tool.md

```diff
@@ -328,8 +328,11 @@ The following documents may help you have a good journey with Prometheus and Gra
 
 [Grafana query metrics from Prometheus](https://prometheus.io/docs/visualization/grafana/#grafana-support-for-prometheus)
 
-Here are two demo pictures of IoTDB's metrics data in Grafana.
+### 5.3. Apache IoTDB Dashboard
+We provide the Apache IoTDB Dashboard, and the rendering shown in Grafana is as follows:
 
-![metrics_demo_1](https://raw.githubusercontent.com/apache/iotdb-bin-resources/main/docs/UserGuide/System%20Tools/Metrics/metrics_demo_1.png)
+![Apache IoTDB Dashboard](https://github.com/apache/iotdb-bin-resources/blob/main/docs/UserGuide/System%20Tools/Metrics/dashboard.png)
 
-![metrics_demo_2](http://raw.githubusercontent.com/apache/iotdb-bin-resources/main/docs/UserGuide/System%20Tools/Metrics/metrics_demo_2.png)
\ No newline at end of file
+You can obtain the json files of Dashboards corresponding to different iotdb versions in the grafana-metrics-example folder.
```

Comment:
I think we can upload the dashboard to [the grafana dashbords](https://grafana.com/grafana/dashboards/)  as the official grafana dashboard of iotdb, it will more conveniently for users to learn about, download and use our dashboard. 
For iotdb, it also provides a place to maintain all versions of dashboards.
Additionally, it is sth like advertisement for iotdb :)

Reply:
Ok, I will check. Thanks for your suggestion.

Reply:
I have already uploaded the dashboard.